code/modules/projectiles/guns/energy.dm

```diff
@@ -214,6 +214,7 @@
 	charge_icon = "+taser"
 	black_market_value = 20
 	actions_types = list(/datum/action/item_action/taser/change_mode)
+	flags_gun_features = GUN_CANT_EXECUTE
```

Comment:
You need to retain the gun features from parent. So either

`flags_gun_features = GUN_UNUSUAL_DESIGN|GUN_CAN_POINTBLANK|GUN_CANT_EXECUTE`
or
`flags_gun_features = parent_type::flags_gun_features | GUN_CANT_EXECUTE`
But I will note its not the norm to do the latter at least yet (515 feature).